src/cbor.h

```diff
@@ -230,8 +230,16 @@ struct CborEncoder
         CborEncoderWriteFunction writer;
     } data;
     uint8_t *end;
+#ifdef __i386__
+    uint32_t remaining;
+    int flags;
+#elifdef __x86_64__
```

Comment:
`#elifdef` is C23. We can't have that.

Reply:
> x86-64 systems without L3 still running? With low RAM that can't deal with a 32-byte struct in modest quantities? Sorry, but I don't buy that.

@thiagomacieira, I building and create PR with patch from Celeron D220 from 2008 without L3 cache with 4 GB DDR2.
I also have MODERN from AMD A6-9225 without L3 cache from 2018 and Intel Atom Z3735F without L3 cache from 2016.

They are all 64-bit and do not have an L3 cache. They have enough RAM for modern tasks, it's just that structs is growing in size and no one project is optimizing it.

Reply:
AMD A6-9225 from 2018, have AVX2 but not have L3 cache. These CPU in my list are unique in their own way, because CPU power consumption.